flow/include/flow/swift_support.h

```diff
@@ -59,11 +59,6 @@
 
 #define SWIFT_STRINGIFY(x) #x
 
-/// Specify that the declared type conforms to the given Swift protocol when the type is
```

Comment:
Where is `SWIFT_CONFORMS_TO_PROTOCOL` coming from? We'll need a loop definition below when Swift is disabled I guess?

Reply:
It's coming from the `<swift/bridging>` header that's part of the Swift toolchain. However, good point, we should not apply it when swift isn't enabled. I will update the patch!